.github/workflows/linux-x64-gpu.yml

```diff
@@ -27,14 +27,30 @@ permissions:
 jobs:
   cuda-118:
     runs-on: ubuntu-latest
-    container: openmmlab/lmdeploy-builder:cuda11.8
     steps:
+      - name: Free disk space
```

Comment:
could we use this action and update docker.yml as well?
https://github.com/InternLM/lmdeploy/blob/b21239a8a3a5a33f3aed98cec7f22e73c0cb9ac9/.github/workflows/docker.yml#L32